omod/src/main/java/org/openmrs/module/adminui/page/controller/myaccount/ChangeDefaultsPageController.java

```diff
@@ -68,6 +79,35 @@ public UserDefaults getUserDefaults(@RequestParam(value = "defaultLocale", requi
     	return new UserDefaults(defaultLocale, proficientLocales);
     }
 
+    public static Locale normalizeLocale(String localeString) {
```

Comment:
Any reason you can't use org.openmrs.util.LocaleUtility.java?

On Thu, Feb 18, 2016 at 9:37 PM, Parth Kachchhi <notifications@github.com>
wrote:

> In
> omod/src/main/java/org/openmrs/module/adminui/page/controller/myaccount/ChangeDefaultsPageController.java
> <https://github.com/openmrs/openmrs-module-adminui/pull/33#discussion_r53424621>
> :
>
> > @@ -68,6 +79,35 @@ public UserDefaults getUserDefaults(@RequestParam(value = "defaultLocale", requi
> >      	return new UserDefaults(defaultLocale, proficientLocales);
> >      }
> >
> > +    public static Locale normalizeLocale(String localeString) {
>
> Let me add a comment on the method. The method is required to convert
> locale string to actual Locale object.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openmrs/openmrs-module-adminui/pull/33/files#r53424621>
> .
>